autopush/router/gcm.py

```diff
@@ -75,6 +75,7 @@ def _route(self, notification, router_data):
             data['body'] = notification.data
             data['con'] = notification.headers['content-encoding']
             data['enc'] = notification.headers['encryption']
+            data['id'] = notification.version
```

Comment:
It's fair to say that APNs is on "hold" for the time being. It's equally fair to say we should remove APNs support until such time as it's requested, then revisit the APNs bridge system.